src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/Claim.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+public interface Claim {
+
+    boolean hasResponsePending();
+
+    boolean hasResponsePendingOverdue();
+
+    boolean hasResponseDueToday();
+
+    boolean hasResponseFullAdmit();
+
+    boolean defendantRespondedWithFullAdmitAndPayImmediately();
```

Comment:
you are probably right but i based it on readability. can we keep them as is? it would be difficult to change because my tests are based on the method names too